src/libcore/ops.rs

```diff
@@ -943,11 +943,20 @@ pub trait IndexMut<Index: ?Sized> {
 }
 
 /// An unbounded range.
+#[cfg(stage0)]
 #[derive(Copy, Clone, PartialEq, Eq)]
 #[lang="full_range"]
 #[unstable = "may be renamed to RangeFull"]
 pub struct FullRange;
 
+/// An unbounded range.
+#[cfg(not(stage0))]
+#[derive(Copy, Clone, PartialEq, Eq)]
+#[lang="range_full"]
+#[unstable = "may be renamed to RangeFull"]
```

Comment:
cc @aturon, would you be comfortable marking this `#[stable]`? (I'd be fine with it)